app/src/main/java/de/tum/in/tumcampusapp/component/ui/overview/CardsRepository.kt

```diff
@@ -74,8 +75,12 @@ class CardsRepository(private val context: Context) {
         }
 
         providers.forEach { provider ->
-            val cards = provider.getCards(cacheControl)
-            results.addAll(cards)
+            try {
+                val cards = provider.getCards(cacheControl)
+                results.addAll(cards)
```

Comment:
please add the reasoning of the PR as a comment

Reply:
Done